examples/gno.land/p/demo/ownable/ownable_test.gno

```diff
@@ -11,57 +11,69 @@ var (
 )
 
 func TestNew(t *testing.T) {
-	std.TestSetOrigCaller(firstCaller)
+	std.TestSetRealm(std.NewUserRealm(firstCaller))
+	std.TestSetOrigCaller(firstCaller) // TODO(bug): should not be needed
 
-	result := New()
-	if firstCaller != result.owner {
-		t.Fatalf("Expected %s, got: %s\n", firstCaller, result.owner)
+	o := New()
+	got := o.Owner()
+	if firstCaller != got {
+		t.Fatalf("Expected %s, got: %s", firstCaller, got)
 	}
 }
 
-func TestOwner(t *testing.T) {
-	std.TestSetOrigCaller(firstCaller)
+func TestNewWithAddress(t *testing.T) {
+	o := NewWithAddress(firstCaller)
+
+	got := o.Owner()
+	if firstCaller != got {
+		t.Fatalf("Expected %s, got: %s", firstCaller, got)
+	}
+}
 
-	result := New()
-	resultOwner := result.Owner()
+func TestOwner(t *testing.T) {
+	std.TestSetRealm(std.NewUserRealm(firstCaller))
```

Comment:
Does `std.NewUserRealm` exist currently? First time I'm seeing it

Reply:
It does; CI is green.

Reply:
I see. It was added in a recent PR, I opened up an [issue](https://github.com/gnolang/gno/issues/2273) for adding it to the docs.